rocketmq-docker/4.2.0/rocketmq-base/scripts/runserver-customize.sh

```diff
@@ -36,7 +36,15 @@ export CLASSPATH=.:${BASE_DIR}/conf:${CLASSPATH}
 #===========================================================================================
 # JVM Configuration
 #===========================================================================================
-JAVA_OPT="${JAVA_OPT} -server -Xms256m -Xmx256m -Xmn128m -XX:MetaspaceSize=128m -XX:MaxMetaspaceSize=320m"
+# Get the max heap used by a jvm, which used all the ram available to the container.
+MAX_POSSIBLE_HEAP=$(java -XX:+UnlockExperimentalVMOptions -XX:MaxRAMFraction=1 -XshowSettings:vm -version |& awk '/Max\. Heap Size \(Estimated\): [0-9KMG]+/{ print $5}' | gawk -f to_bytes.gawk)
+
+# Dynamically calculate parameters, for reference.
+Xms=$[MAX_POSSIBLE_HEAP/4]
```

Comment:
Same as the above comments.

Reply:
@huanwei what's the difference between runserver and runbroker

Reply:
@vongosling There are several different JVM options between `${ROCKETMQ_HOME}/bin/runserver.sh` and `${ROCKETMQ_HOME}/bin/runbroker.sh`.

Reply:
@huanwei Do we have to have two configurations？

Reply:
@vongosling Yeah. Currently I think so. I investigated the apache/rocketmq structure and found there are many different jvm configurations between the above two scripts. So at least for now I think we have to keep the two, one for broker and the other one for namesrv.

Reply:
@vongosling Check below for differences.
```
[runbroker.sh](https://github.com/apache/rocketmq/blob/master/distribution/bin/runbroker.sh)
[runserver.sh](https://github.com/apache/rocketmq/blob/master/distribution/bin/runserver.sh)
```